test/test_testing.py

```diff
@@ -2389,30 +2389,30 @@ def test_get_pebble_container_plan(self):
         self.assertEqual(initial_plan.to_yaml(), '{}\n')
         container = harness.model.unit.get_container('foo')
         container.pebble.add_layer('test-ab', '''\
-summary: test-layer
-description: a layer that we can use for testing
-services:
-  a:
-    command: /bin/echo hello from a
-  b:
-    command: /bin/echo hello from b
-''')
+            summary: test-layer
```

Comment:
It's a bit hard to see what changed here -- would it be better to move the dedenting (at least for existing code) to another PR to keep this diff small and clear?